WalletWasabi.Fluent/Controls/LabelsItemsPresenter.axaml

```diff
@@ -49,7 +49,8 @@
     </Setter>
     <Setter Property="ItemsPanel">
       <ItemsPanelTemplate>
-        <c:LabelsPanel Orientation="Horizontal" Spacing="2" HorizontalAlignment="{TemplateBinding HorizontalAlignment}">
+        <c:LabelsPanel Orientation="Horizontal" Spacing="2"
+                       HorizontalAlignment="{Binding $parent[c:LabelsItemsPresenter].HorizontalAlignment}">
```

Comment:
Just curious why didn't the previous one work? What is the difference?

Reply:
TemplateBinding only works inside control templates.